app/lib/code_red.rb

```diff
@@ -54,7 +54,8 @@ def create_contact_for_post(subscription)
         'HomeEmail' => subscription.email,
         'FirstName' => subscription.first_name,
         'LastName' => subscription.last_name,
-        'HomePhone' => subscription.call ? subscription.phone_number : '',
+        'OtherPhone' => subscription.phone_number,
+        #'HomePhone' => subscription.call ? subscription.phone_number : '',
```

Comment:
What's the reason behind this change? Wouldn't this mean that you'd get both a text and a call, since you removed the conditional?

Also, typically don't leave commented-out code around, especially without an explanation for why it's both commented out and not deleted entirely.